### ISSUE TITLE
RHEL and CentOS installers?

### ISSUE THREAD

**Issue by talkingmoose** [jamf/JAWA#9]
Can you post the installers for RHEL and CentOS?

```         __  
  _      \ \ 
 (_)______| |
   |______| |
  _       | |
 (_)      | |
         /_/

**Comment by matthewsphillips** [comment]
That is on the Roadmap. ;)

**Comment by dkmansion** [comment]
How bout the source for the ubuntu installer to reverse engineer ;) for rhel/centos creation?  Could contribute :)

**Comment by ball42** [comment]
The rhel_installer.sh included in v3.0.1 was also tested on CentOS.  Since CentOS is rapidly approaching end of life (Dec 31, 2021) will be done on Rocky Linux and/or AlmaLinux in the future.  ٩(◕‿◕｡)۶
